rel/post_start/migrate.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+release_remote_ctl eval --mfa "Mix.Tasks.Archethic.Migrate" --argv "$@"
```

Comment:
An error is returned when script is called :
`▸  Incomplete module/function/arity specification for --mfa!: "Mix.Tasks.Archethic.Migrate"`
In the command you need to pass the function name `"Mix.Tasks.Archethic.Migrate.run/1"`